src/types.ts

```diff
@@ -0,0 +1,34 @@
+import { z } from 'zod';
+import { EvmAddress, evmAddressSchema, EvmId, evmIdSchema } from './config/schema';
+
+export type AirnodeAddress = EvmAddress;
+export type TemplateId = EvmId;
+
+// Taken from https://github.com/api3dao/signed-api/blob/main/packages/api/src/schema.ts
+// TODO should be imported
```

Comment:
Honestly, I'd not bother. Making these two services coupled will cause more complexity then advantages.

But I am curious how do we plan to handle compatibility between these systems. That seems out of the scope of this PR though.